tests/support/callbacks_for_tests.py

```diff
@@ -74,3 +75,12 @@ def callback_with_sleep_three_seconds(payload: Payload):
     payload.ack()
     logger = logging.getLogger(__name__)
     logger.info("%s sucessfully processed!", payload.body)
+
+
+def callback_with_explicit_db_connection(payload: Payload):
+    Simple.objects.create()
+
+    thread = threading.current_thread()
+    setattr(thread, "db", db.connections["default"])
```

Comment:
**suggestion (non-blocking):** do you think it's worth commenting that this `db.connections` object stores thread-isolated connections, which can be accessed with a [dict-like](https://github.com/django/django/blob/ca9872905559026af82000e46cde6f7dedc897b6/django/utils/connection.py#L56) syntax, and is being used here to add this connection as new attribute to the current thread to perform tests later?

Reply:
Sure thing! I added a docstring in this callback ([b30bd78](https://github.com/juntossomosmais/django-stomp/pull/44/commits/b30bd78be380825f9aa34acf6a546b2580374bad))

Reply:
Wonderful notes, my friend. Thanks!